app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -33,6 +33,23 @@ def deny_access(exception)
     end
   end
 
+  def flag
+    curation_concern.state = 'flagged'
+    note = params[curation_concern_name][:workflow_note]
+    curation_concern.workflow_note = curation_concern.workflow_note + [note] unless note.blank?
```

Comment:
Oh. Yeah. https://github.com/ActiveTriples/ActiveTriples/issues/19 =( I should have looked at the code, this has been an issue for a long time.